src/public/js/video-tagging/optional-tags.html

```diff
@@ -92,12 +94,11 @@
                 else if (this.tagsState[tag] === 'empty' || this.tagsState[tag] === 'partial') {
                     this.tagsState[tag] = "full";
                     self.setSelected(e.currentTarget);
-                    self.fire("ontagschanged", { selected: this.projectSelectedTags(), state: this.tagsState});
                 } else {
                     this.tagsState[tag] = "empty";
                     self.setUnselected(e.currentTarget);
-                    self.fire("ontagschanged", { selected: this.projectSelectedTags(), state: this.tagsState});
-                }         
+                } 
+                self.fire("ontagschanged", { selected: this.projectSelectedTags(), state: this.tagsState});        
```

Comment:
Expected to now always fire vs. previously was only firing event in the else case.